contents/docs/product-analytics/lifecycle.mdx

```diff
@@ -28,19 +42,51 @@ Lifecycle breaks down whether a user has been active in your selected time inter
 To start with, you'll need to select an event that best describes activity for your product, this might be something as simple as logging into your app or loading the page. For example: in a messaging product, you might consider someone active if they've sent a message within the interval.
 
 You can see here that I have selected the Message Sent event to define my user activity metric:
-![image](https://user-images.githubusercontent.com/85295485/139204932-dfd438d7-cb86-407c-8a0e-efb6a906f3d3.png)
+
+<ProductScreenshot
+    imageLight = {sentAMessageLight} 
+    imageDark = {sentAMessageDark}
+    classes="rounded"
+    alt="Set up a new lifecycle analysis with custom event 'Sent a message'"
+/>
 
 As with other tools in PostHog, you can add filters if you wish to focus on a subset of these events.
 
-![image](https://user-images.githubusercontent.com/85295485/139205269-21790a9d-6f4c-471a-ae0a-76b2b6557646.png)
+<ProductScreenshot
+    imageLight = {lifecycleFiltersLight} 
+    imageDark = {lifecycleFiltersDark}
+    classes="rounded"
+    alt="Add filters for lifecycle analysis"
+/>
+
+Next set the time interval that aligns with the natural frequency of your product by updating the "grouped by" setting. Here were' looking at "Weekly" active as in this hypothetical scenario, we expect people to be sending messages every week, but depending on how people use your product, you might want to change the group by setting to by "Hour", "Day" or "Month". You can also set how far back you would like to consider this data over.
 
-Next set the time interval that aligns with the natural frequency of your product, here were' looking at "Daily" active since we expect people to be sending messages every day, but depending on how people use your product, you might want to change the interval to "Week" or "Month". You can also set how far back you would like to consider this data over.
+<ProductScreenshot
```

Comment:
The text above groups by week but shows "day" highlighted and selected, could we fix this?